src/app/components/nav/Nav.js

```diff
@@ -4,13 +4,14 @@ import { NavLink } from "react-router-dom";
 import Container from "react-bootstrap/Container";
 import Navbar from "react-bootstrap/Navbar";
 import Nav from "react-bootstrap/Nav";
+import ExternalLink from "../../containers/external-link/ExternalLink";
```

Comment:
Can we use an absolute import instead of traversing the parent directories? e.g. `from "containers/..."`